test/extended/testdata/olmv1/install-operator.yaml

```diff
@@ -28,7 +28,8 @@ spec:
       name: install-test-sa
   source:
     catalog:
-      packageName: quay-operator
+      packageName: {PACKAGENAME}
+      version: {VERSION}
```

Comment:
If you go through another iteration, perhaps replace `{REPLACE}` with `{NAMESPACE}`?